chains/evm/contracts/test/ccvs/CommitteeRamp/CommitteeRamp.getDynamicConfig.t.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {CommitteeRamp} from "../../../ccvs/CommitteeRamp.sol";
+import {CommitteeRampSetup} from "./CommitteeRampSetup.t.sol";
+
+contract CommitteeRamp_getDynamicConfig is CommitteeRampSetup {
```

Comment:
nit: generally I don't write separate files for getters that are already fully covered by the test files of the setters. Can keep for now but imo no need in the future